### ISSUE TITLE
Setting etcd host and port

### ISSUE THREAD

**Issue by amalagaura** [tagplay/docker-hubot#4]
Just a comment on the HUBOT_ETCD_HOST and HUBOT_ETCD_PORT. The following does not work:

```
      - HUBOT_ETCD_HOST=etcd
      - HUBOT_ETCD_PORT=2379
```

`ERROR cannot connect to etcd cluster: etcd`

The following **does** work:

```
      - HUBOT_ETCD_HOST=http://etcd
      - HUBOT_ETCD_PORT=2379
```

or

```
      - HUBOT_ETCD_HOST=http://etcd:2379
```

So it seems silly to have the separate PORT variable. It seems a lot easier and simpler if we would just say:

```
      - HUBOT_ETCD_URL=http://etcd:2379
```

And just pass that directly on the command line instead of creating the NODE variable in the script.

**Comment by amalagaura** [comment]
Can I create a pull request for this?

It will reverse the first pull request which created the variable. That person can accomplish the same thing by only setting the URL variable without a port. The downside is that it is a breaking change.

**Comment by geowa4** [comment]
I agree we should only need to set `HUBOT_ETCD_URL`. I would support that PR.
